packages/react-native/src/primitives/TextField/TextField.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useMemo } from 'react';
+import { TextInput, View, ViewStyle } from 'react-native';
+import { Label } from '../Label';
+
+import { styles } from './styles';
+import { TextFieldProps } from './types';
+
+export default function TextField({
+  accessibilityLabel,
```

Comment:
Do we need `accessibilityRole` as well?

Reply:
I haven't come across an accessibilityRole for input